resources/public/cheatSheet.html

```diff
@@ -0,0 +1,65 @@
561%1F7Tp1%1FVpiz%1FAgnd%1F0118700561%1FE1900%1FB1997%1FaHager%1FdWerner%1E033A+%1FpLeipzig%1E034D+%1Fa82+S.%1E034I+%1Fa4%1E037A+%1FaAuch+in+Buchh.+als%3A+Ro%CC%88mische+Forschungen%2C+hrsg.+v.+d.+Bibliotheca+Hertziana+in+Rom.+Bd+7.+1929.%1E037C+%1FaBasel%2C+Phil.-+hist.+Diss.%1E%0A001@+%1F01-2%1Fa5%1E001A+%1F01200%3A28-09-87%1E001B+%1F09999%3A29-12-07%1Ft19%3A41%3A36.000%1E001D+%1F09999%3A99-99-99%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Aan%1E003@+%1F0871159031%1E004A+%1Ffgeh.+%28Pr.+nicht+mitget.%29%1E006U+%1F087%2CB42%2C0848%1E007I+%1FSo%1F0310869122%1E011@+%1Fa1987%1E021A+%1FaEinfach+klassisch%21%1FdEine+Orestie+mit+glu%CC%88ckl.+Ausgang%1FhWalter+Mehring+%281896+-+1981%29.+Mit+e.+Nachw.+hrsg.+von+Didier+Plassard.+Univ.+-+Gesamthochsch.+Siegen%1E028A+%1F9118579983%1F7Tp1%1FVpip%1FAgnd%1F0118579983%1FE1896%1FB1981%1FaMehring%1FdWalter%1E032@+%1Fg12%1Fa%282.+Aufl.%29%1E033A+%1FpSiegen%1FnUniv.+-+Gesamthochsch.%1E034D+%1Fa28+S.%1E034I+%1Fa21+cm%1E036F+%1Fx216%1F9010243046%1FgAdvz%1Fi19442-6%1FYVergessene+Autoren+der+Moderne%1FpSiegen%1FJUniv.-Gesamthochschule%1Fl16%1E045E+%1Fa59%1Fm810%1Fm820%1Fm830%1Fm839%1Fm840%1Fm850%1Fm860%1Fm870%1Fm880%1Fm890%1FmB%1E047A+%1FSFE%1Fabs%1E%0A001@+%1F01%1Fa5%1E001A+%1F09999%3A31-10-01%1E001B+%1F00101%3A07-08-09%1Ft17%3A28%3A39.000%1E001D+%1F09999%3A31-10-01%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Aal%1E003@+%1F0572736959%1E006U+%1F000%2CL01%1E007I+%1FSo%1F072019536%1E007M+%1F02797/0022%1E010@+%1Fager%1E011@+%1Fa1916%1E021A+%1FaDorfheimat%1FdBilder+aus+der+Knabenzeit%1Fhvon+Franziskus+Nagler%1E028A+%1F9116880430%1F7Tp1%1FVpiz%1FAgnd%1F0116880430%1FE1873%1FB1957%1FaNagler%1FdFranciscus%1E032@+%1Fg13%1Fa3.+Aufl.%1E033A+%1FpMeissen%1FnBuchheim%1E034D+%1Fa208+S.%1E034I+%1Fa8%1E037A+%1FaIn+Fraktur%1E047E+%1Faagi-809Haerig%1E%0A001@+%1Fa5%1E001A+%1F01145%3A29-03-04%1E001B+%1F01145%3A27-04-05%1Ft18%3A02%3A05.000%1E001D+%1F09999%3A10-05-08%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Acl%1E003@+%1F0560568487%1E007I+%1FSo%1F073290854%1E007M+%1F01769/0218%1E017A+%1Fagw%1E021A+%1FaMein+Hermannsland%1E033A+%1FpBielefeld%1FpLeipzig%1FnVelhagen+%26+Klasing%1E034I+%1Fa8%1E036G+%1FaVelhagen+%26+Klasings+Erga%CC%88nzungshefte+zu+deutschen+Lesebu%CC%88chern%1E042@+%1F0NN%1Fc34p3_vk.dat%1Fd1337%1Fe2004-03-29%1E%0A001@+%1F01-2%1Fa5%1E001A+%1F01130%3A27-10-05%1E001B+%1F09999%3A23-11-05%1Ft10%3A44%3A17.000%1E001D+%1F01140%3A03-11-05%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Af%1E003@+%1F0976858223%1E004A+%1Ffkart.%1E006U+%1F005%2CB49%2C0021%1E007I+%1FSo%1F0723555963%1E011@+%1Fa2005%1Fn%5B2005%5D%1E017A+%1Farb%1E021A+%1Fx13%1F9957878923%1FYSoftware-Architektur%1FhFernuniversita%CC%88t+in+Hagen%1FpHagen%1FJFernuniv.%1FS004%1FgAc%1E021B+%1FlKurseinheit+3.%1FaGenerische+Architekturen+und+Komponentensysteme%1FhAutor%3A+A.+Poetzsch-Heffter.+Mitarb.%3A+N.+Rauch%1E032@+%1Fg16%1Fa%5B6.+Aufl.%5D%1E034D+%1FaS.+66+-+102%1E034I+%1FaIll.%1E047A+%1FSFE%1Fa%5BADH%1E047A+%1FSERW%1Farm%1E&active-editor=fix">Count occurences of values of a specified field</a>
```

Comment:
This is already available in the examples: https://metafacture.org/playground/?example=Count_values_for_path